relevanceai/datasets.py

```diff
@@ -480,6 +473,13 @@ def get_machine_learning_research_dataset():
     raise NotImplementedError
 
 
-get_dummy_ecommerce_dataset = get_ecommerce_1_dataset
-get_sample_ecommerce_dataset = get_ecommerce_2_dataset
-get_ecommerce_dataset = get_ecommerce_3_dataset
+### For backwards compatability

```

Comment:
@boba-and-beer  Hello! I have resolved this here, however if you want me instead to keep fn name as `get_ecommerce_1_dataset`, `get_ecommerce_2_dataset` etc pls let me know!